pkg/restic/exec_commands.go

```diff
@@ -47,14 +47,8 @@ type backupStatusLine struct {
 // GetSnapshotID runs a 'restic snapshots' command to get the ID of the snapshot
 // in the specified repo matching the set of provided tags, or an error if a
 // unique snapshot cannot be identified.
-func GetSnapshotID(repoIdentifier, passwordFile string, tags map[string]string, env []string, caCertFile string) (string, error) {
-	cmd := GetSnapshotCommand(repoIdentifier, passwordFile, tags)
-	if len(env) > 0 {
-		cmd.Env = env
-	}
-	cmd.CACertFile = caCertFile
-
-	stdout, stderr, err := exec.RunCommand(cmd.Cmd())
+func GetSnapshotID(snapshotIdCmd *Command) (string, error) {
```

Comment:
Minor: there's no longer tags being passed in and also this function signature changed enough that making a change to the godoc would probably be useful.